### DISCUSSION TITLE
Next release on pypi.org

### DISCUSSION THREAD

**Discussion by Shqrk** [rytilahti/python-miio#1308] [General]
Hey,

I would like to know when will be the next release on https://pypi.org/project/python-miio/ ?

Thanks in advance

**Comment by Shqrk** [comment]
Hey,

Have you any information ?

Thanks you

**Comment by rytilahti** [comment]
Hi @Shqrk, thanks for notifying me about this, I'll make a new release by the end of this week!

**Comment by rytilahti** [comment]
0.5.10 has now been released https://github.com/rytilahti/python-miio/releases/tag/0.5.10 and is also available on pypi!

**Reply by Shqrk** [reply]
Thanks you very much :)
